components/src/core/selection-toolbar/selection-toolbar.component.scss

```
@@ -0,0 +1,40 @@
+.pxb-selection-toolbar-icon-wrapper {
+    display: flex;
+    align-items: center;
+    z-index: 1;
+    margin-right: 24px;
+    &.pxb-selection-toolbar-no-icon {
+        margin-right: 0;
+    }
+}
+
+.pxb-selection-toolbar-title-container {
+    display: flex;
+    padding-right: 16px;
+    justify-content: center;
+    align-self: stretch;
+    flex-direction: column;
```

I think you can simplify this (and make it more browser compatible) by just using display block:
```suggestion
    display: block;
    padding-right: 16px;
```